applications/GeoMechanicsApplication/custom_utilities/process_utilities.cpp

```diff
@@ -45,4 +73,30 @@ std::vector<std::reference_wrapper<ModelPart>> ProcessUtilities::GetModelPartsFr
     return result;
 }
 
+void ProcessUtilities::AddProcessesSubModelPartList(const Parameters& rProjectParameters, Parameters& rSolverSettings)
```

Comment:
To make it really clear what this function does, I'd propose to add to the name that we add the list to the solver settings:
```suggestion
void ProcessUtilities::AddProcessesSubModelPartListToSolverSettings(const Parameters& rProjectParameters, Parameters& rSolverSettings)
```

Reply:
Indeed, it is clearer.